src/content/docs/about/getting-started.mdx

```diff
@@ -0,0 +1,210 @@
+
+## Using the CLI
+
+The Daytona Command-Line Interface (CLI) is the primary method of interacting with the Daytona platform. The CLI provides a comprehensive set of commands that allow you to manage Workspaces, Providers, and other system configurations.
+
+To access the Daytona CLI, open your terminal and use the `daytona` command followed by the desired subcommand. The CLI offers a wide range of commands, each serving a specific purpose, such as creating Workspaces, managing Providers, setting Targets, and configuring system settings.
+
+For a complete list of available commands and their descriptions, refer to the [CLI Reference](/reference/cli). This reference guide provides detailed information on each command, including usage examples and command options, helping you leverage the full potential of the Daytona CLI.
+
+Additionally, you can use the `daytona --help` command to list all commands, offering a convenient way to explore available options.
+
+<DocumentList>
+<DocumentListItem
+    title="CLI Reference"
+    subtitle="Learn how to use the Daytona Command-Line Interface (CLI) to manage your development environments."
+    href="/reference/cli"
+    expanded
+  />
+</DocumentList>
+
+## Adding a Git Provider
+
+You can integrate Daytona with a Git Provider to manage version control operations and interact with your Git repositories.
+
+Daytona supports a variety of [Git Providers](/configuration/git-providers), including [GitHub](/configuration/git-providers#github), [GitLab](/configuration/git-providers#gitlab), [Bitbucket](/configuration/git-providers#bitbucket), and self-hosted solutions. This integration supports and enables interacting with your codebase, cloning repositories, and pushing changes directly from Daytona.
+
+<br />
+
+<CardGrid gridNumber={3} mobileGridNumber={1} colsMinWidth='150px' colsMinWidthMobile='100px'>
+    <TitleCard icon={githubLogo} title="GitHub" link="/configuration/git-providers#github" imageAlt="'GitHub logo"/>
+    <TitleCard icon={gitlabLogo} title="GitLab" link="/configuration/git-providers#gitlab" imageAlt="GitLab logo"/>
+    <TitleCard icon={bitbucketLogo} title="Bitbucket" link="/configuration/git-providers#bitbucket" imageAlt="BitBucket logo"/>
+</CardGrid>
+
+Use the `daytona git-providers --help` command to view available options for managing your Git Provider configurations and interacting with your repositories.
+
+<DocumentList>
+<DocumentListItem
+    title="Add a Git Provider"
+    subtitle="Learn how to add a Git Provider to Daytona and start managing your repositories effectively."
+    href="/configuration/git-providers#add-a-git-provider"
+    expanded
+  />
+</DocumentList>
+
+## Installing a Provider
+
+A [Provider](/configuration/providers) is a plugin that interfaces with the [Daytona Server](/usage/server), handling the deployment of the development environment and its lifecycle of management.
```

Comment:
Took the liberty of trying to definde providers better, @Tpuljak please give your thoughts:

Providers are the mechanisms through which Daytona integrates with various technologies to create and manage development environments. These providers serve as the foundational engines that Daytona.io leverages to deploy and run your environments, whether through containerization, orchestration, or cloud-based virtual machines. By abstracting the complexities of these underlying technologies.

Reply:
I like it. Minor fixup:
```
Providers are plugins through which Daytona integrates with various technologies to create and manage development environments. These providers abstract complexities of underlying technologies and serve as the foundational engines that Daytona leverages to deploy and run your environments, whether through containerization, orchestration, or cloud-based virtual machines.
```